php-classes/Slate/CBL/StudentCompetency.php

```diff
@@ -13,6 +13,8 @@
 class StudentCompetency extends \ActiveRecord
 {
     public static $autoGraduate = true;
+    public static $isLevelComplete;
+    public static $minimumRatingOffset = null;
```

Comment:
@themightychris There is a strict comparison against minimumRatingOffset on line 342, do I need to declare it as null here or can I simply leave the value blank?

Reply:
@jtwiest `null` is the default value for an uninitialized class property, explicitly setting it `= null` is not necessary